config/v1/types_image.go

```diff
@@ -112,4 +112,7 @@ type RegistrySources struct {
 	// Only one of BlockedRegistries or AllowedRegistries may be set.
 	// +optional
 	AllowedRegistries []string `json:"allowedRegistries,omitempty"`
+	// unqualifiedSearchRegistries are registries that will be searched when pulling images using short names.
```

Comment:
since these doc comments end up as `oc explain` i would really to see 

- what are short names,
- can example of the setting and when we used with a short name what behavior is expected.
- comment on the order or priority wrt search order, are these in decreasing priority or container engine will decide that on its own
- how do i know if the setting has been applied to the cluster and i can use this feature now.
- any restrictions on these values like you can not have registries that are already set in blocked registries or can not set the registry that hosts the release image maybe etc..

Reply:
nit: `unqualifiedSearchRegistries` for me this word has very little meaning i.e when seeing this field in documentation is is not quickly apparent to me what this would do.. just saying as someone who is not completely in know of the registry stuff.. maybe that is okay as you need that background context as you are editing the Image object.

Reply:
@abhinavdahiya `unqualifiedSearchRegistries` corresponds directly to the configuration option in `registries.conf` [1] that is used by podman. I think it is fair to assume that a customer altering this option is familiar with the RHEL/podman equivalents.

With regard to "short names" - I suggest the following:

```
unqualifiedSearchRegistries are registries that will be searched when pulling images that do not have fully qualified domains in their pull specs. Registries will be searched in the order provided in the list.
```

[1] https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md

Reply:
Fixed with suggestion.